x/concentrated-liquidity/keeper_test.go

```diff
@@ -26,6 +26,10 @@ import (
 )
 
 var (
+	// TODO: switch:
```

Comment:
Might be too late for this but I wonder if it would have been beneficial to have a special TODO for precision stuff to make it easier to be sure we don't miss any of these later (e.g. `TODO (precision):` so we can search for that when #5726 is done)

Reply:
Yeah, that's a good idea!

However, I made sure that every single one is tracked in the epic though so not worried about misses:
https://github.com/osmosis-labs/osmosis/issues/5864